### PR TITLE
Document how to get a single package from nixpkgs-unstable

### PR DESCRIPTION
**Pull request by zupo**

### CODE REVIEW

File: docs/common-patterns.md [hunk 1]
```diff
@@ -1,4 +1,29 @@
...
+
+{
+  packages = [
+    pkgs-unstable.elmPackages.elm-test-rs
+  ];
+}
+```
+
 
 ### In the file `devenv.yaml` edit the `nixpkgs` input to point to nixpkgs-unstable
```
Comment (domenkozar) [inline docs/common-patterns.md:28]: this part should be removed

Comment (zupo) [reply docs/common-patterns.md]: Ah yes good catch.

File: docs/common-patterns.md [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+```
+
+2. Use the package in your `devenv.nix`:
+
+```nix
+{ ... }:
+
+{
+  packages = [
+    pkgs-unstable.elmPackages.elm-test-rs
```
Comment (domenkozar) [inline docs/common-patterns.md:22]: I'm not sure this works, there's no pkgs-unstable in scope here

Comment (zupo) [reply docs/common-patterns.md:22]: 🤦 you're right, fixing

Comment (zupo) [reply docs/common-patterns.md:24]: done

**Pull request merged**